src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -604,6 +605,7 @@ private function addAnnotationsSection(ArrayNodeDefinition $rootNode)
                     ->info('annotation configuration')
                     ->canBeDisabled()
```

Comment:
Hum...The two codes slightly differ. With the patch above:

```php
    annotations:
        cache: php_array
```

will properly enable annotations whereas the current implementation doesn't (which is wrong).

So I'll apply the patch.